src/Presentation/Presenters/DonationConfirmationHtmlPresenter.php

```diff
@@ -19,13 +19,14 @@ public function __construct( TwigTemplate $template ) {
 		$this->template = $template;
 	}
 
-	public function present( Donation $donation, string $updateToken ): string {
-		return $this->template->render( $this->getConfirmationPageArguments( $donation, $updateToken ) );
+	public function present( Donation $donation, string $updateToken, string $templateName = '' ): string {
```

Comment:
What's the reasoning behind switching this template included in the main one, rather than just switching the main one? The later case seems more straightforewards to me, as it avoids dealing with two templates, and having this $templateName thing here that can be empty, and then default to something, but only if the provided main template takes care of that.